presto-main/src/main/resources/webapp/src/components/SQLClient.jsx

```diff
@@ -340,9 +82,22 @@ export default function SQLClientView() {
             <div className='container'>
                 <div className="row">
                     <div className="col-xs-12">
-                        <h3>SQL</h3>
+                        <ul className="nav nav-tabs">
+                            {views.map((view, idx) => (
+                                <li key={idx} className={clsx(values.view === view.name && 'active')}>
```

Comment:
```suggestion
                                <li key={idx} className={clsx({active: values.view === view.name})}>
```

Sorry to keep bugging about this, the idea of `clsx` is that you can pass an object to it, the keys are the classes you want to add, and the value is the condition on which they should apply.

Here is a link to the usuage options https://www.npmjs.com/package/clsx#usage .

Reply:
I believe I am using the `Strings (variadic)` :-)

Reply:
THanks, makes sense for a small use case like this 👍